$NetBSD: patch-be,v 1.2 2006/02/18 10:12:24 rillig Exp $

Status: applied upstream.

There's some prototype conflict with SunPro, when the constructor is
called. The "const int" is unnecessary, as ints are always passed by
value.

--- sunos5/cpumeter.h.orig	1999-01-31 21:26:38.000000000 +0100
+++ sunos5/cpumeter.h	2005-02-22 14:16:47.280965600 +0100
@@ -12,7 +12,7 @@
 
 class CPUMeter : public FieldMeterGraph {
  public:
-	CPUMeter(XOSView *parent, kstat_ctl_t *kcp, const int cpuid = 0);
+	CPUMeter(XOSView *parent, kstat_ctl_t *kcp, int cpuid = 0);
 	~CPUMeter(void);
 
 	const char *name(void) const { return "CPUMeter"; }
